backend/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.5'
+services:
+  adminer:
```

Comment:
If I understood correctly this is a tool to manage the the database. Wonder what are the pros of adding it to the docker-compose though? Couldn't we use install it locally on our machines as this is not per se part of our application?